### PR TITLE
Iteration 2 date

### PR DESCRIPTION
**Pull request by drod1000**
removed seeds, updated data type for installation date to date time and updated rake rile so date time works

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -27,17 +25,16 @@
   end
 
   create_table "trips", force: :cascade do |t|
```
Comment (robbie-smith) [inline db/schema.rb:27]: We need the create_table trips in your pull request to be the same as master.

File: Rakefile [hunk 2]
```diff
@@ -22,11 +23,11 @@ def create_trips
     Trip.create(row)
   end
   puts "Imported Trips to Table."
-end
+ end
 
 def create_stations
```
Comment (robbie-smith) [inline Rakefile:26]: Could you update the create_stations method in your branch to be consistent with master. For example :
 ```
SmarterCSV.process('db/csv/station.csv').each do |row|
     Station.create(id: row[0], name: row[1], dock_count: row[4], city: row[5], installation_date: Date.strptime(row[6], "%m/%d/%Y"))
  end
  puts "Imported Stations to Table."
```

**Review by robbie-smith — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 1 above)
Comment (drod1000) [reply db/schema.rb:27]: where is it different? I only changed create stations

Comment (robbie-smith) [reply db/schema.rb:27]: Did you look at view changes?

Comment (drod1000) [reply db/schema.rb:27]: yeah it only shows the stations part as green. and that one I have to change to add date time. what lines are you referring to?

Comment (robbie-smith) [reply db/schema.rb:27]: Hmm, it is I wish I could show you my screen. Maybe I am misreading.

**Pull request closed without merging**